[LTC] Fix some CI issues

alanwaketan opened the pull request:
Summary:
file_diff_from_base() has issues while being run on PRs that are not targeting master.
This patch hacks it to work with lazy_tensor_staging branch.

Commit:
[LTC] Fix some CI issues

Summary:
file_diff_from_base() has issues while being run on PRs that are not targeting master.
This patch hacks it to work with lazy_tensor_staging branch.

seemethere reviewed (approved):
*NOTE*: This _should not_ be merged into master whenever `lazy_tensor` is eventually merged back into `master`

Commit:
[LTC] Fix some CI issues

Summary:
Fixed get_base_commit() to track the head of lazy_tensor_staging instead of the master.
